elyra/pipeline/processor_kfp.py

```diff
@@ -76,7 +78,12 @@ def process(self, pipeline):
                                       pipeline_id=kfp_pipeline.id)
 
             self.log.info("Starting Kubeflow Pipeline Run...")
-            return "{}/#/runs/details/{}".format(api_endpoint, run.id)
+
+            return PipelineProcessorResponse(
+                experiment_url="{}/#/runs/details/{}".format(api_endpoint, run.id),
```

Comment:
I'm not sure what _handle_ would be used to perform restarts or retry failed runs - guessing `pipeline_name` for example - but we should include that in the response since that's what would be used in a subsequent request (PUT /api/scheduler/{pipeline_name}) for retrying or continuing runs.

I'm not suggesting this must be displayed in the response dialog, but it needs to be available for future calls.

Reply:
As we don't have the `experiment list` available on Elyra today, it would be a little hard. KFP is also not a good citizen in this area (at least in my opinion). Having said that, once the user is on the KFP UI, he can manage all that.